openmdao/docs/openmdao_book/features/core_features/working_with_groups/connect.ipynb

```diff
@@ -229,21 +236,130 @@
     "assert_near_equal(p.get_val('C1.x'), np.array([[0., 10.],\n",
     "                                               [7., 4.]]))\n",
     "assert_near_equal(p.get_val('C1.y'), 42.)"
-   ],
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using `om.slicer` to connect non-scalar variables\n",
+    "\n",
+    "The `om.slicer` object was introduced to OpenMDAO to make connecting non-scalar variables easier by allowing the `src_indices` to be specified as slices.\n",
+    "This way of thinking about `src_indices` is probably more intuitive for users who are experienced with numpy.\n",
+    "Note when `src_indices` is provided as a slicer object, `flat_src_indices` has no effect.\n",
```

Comment:
This is no longer true since we refactored how indexing works a while back.  For example, for om.slicer[:1] where the src is a 3x3 array.  With flat_src_indices=False, result after indexing is size 3 (first row of the array), but if flat_src_indices=True, result is size 1 (first entry of the flattened array).

Reply:
Fixed, and added new examples.